qiskit/providers/ibmq/managed/managedjobset.py

```diff
@@ -385,44 +398,44 @@ def job(
 
     @requires_submit
     def qobjs(self) -> List[Qobj]:
-        """Return the Qobj for the jobs.
+        """Return the Qobj's for the jobs in this set.
```

Comment:
The apostrophe could be removed here and below in line `404`

Reply:
There are multiple Qobj's being returned. I think adding `'s` is the proper grammar?